Contour.appdata.xml

```diff
@@ -1,9 +1,99 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<component type="runtime">
+
+<component type="desktop">
   <id>org.contour.Terminal</id>
+  <metadata_license>MIT</metadata_license>
   <project_license>Apache-2</project_license>
+  <content_rating>none</content_rating>
```

Comment:
this is an invalid value

Reply:
I've fixed that as I realized it won't let me finish flatpak-builder even locally. Thanks.